src/api/article/collection/middlewares.js

```diff
@@ -0,0 +1,13 @@
+export const parseLocalized = (req, res, next) => {
```

Comment:
I think it's redundant, cause we already have [`body-parser`](https://github.com/babajka/babajka-backend/blob/master/src/server.js#L19)